2_process/src/process_data.R

```diff
@@ -5,12 +5,16 @@ library(stringr)
 library(sbtools)
 library(whisker)
 
-
+# Read in the data downloaded in the prior step
```

Comment:
This change looks good, Jesse - only edit is that here you want to be reading in the *processed* data (`2_process/out/model_summary_results.csv`), rather than the raw downloaded data